src/components/utility/FormElement/FormElement.jsx

```diff
@@ -2,15 +2,21 @@ import React, { useState } from "react";
 import styles from "./FormElement.module.css";
 import Typography from "../Typography/Typography";
 
-const FormElement = ({ type, placeholder, value, onChange, error }) => {
+const FormElement = ({ type, placeholder, value, onChange, onBlur,onFocus, error }) => {
```

Comment:
Very nitpicky, but please add space between these two (for readability)

Reply:
done